### DISCUSSION TITLE
Password Generator suggestion

### DISCUSSION THREAD

**Discussion by lzrdfrt** [DevToys-app/DevToys#899] [Ideas]
I love the password generator and options for exclusions would be excellent.

An example would be leaving the Include Special Characters option on but having the ability to exclude certain characters like backslashes, single or double quotes and such.  Maybe an additional option to exclude 'look-alike' characters (capital o and zero, lowercase L and 1, etc).

Thank you for a building such a useful tool!

**Comment by Anomerly** [comment]
Yeah, I have systems where I need special characters but certain characters are forbidden, it would be nice to have a menu of the special characters where you could toggle particular characters in and out.
